.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Lint
+
+on: push
+
+jobs:
+  pipeline:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Linter
+        run: pip install black
+        
+  lint:
```

Comment:
Please don't use two stages, use only one stage, as you did in the first commit - 20a0744